src/common-utils/main.sh

```diff
@@ -551,7 +551,11 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
 
         # Add devcontainer .zshrc template
         if [ "$INSTALL_OH_MY_ZSH_CONFIG" = "true" ]; then
-            echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
+            if [ -f "${template_path}" ] && grep -qF "$(head -n 1 "${template_path}")" "${user_rc_file}"; then
+                echo "No need to overwrite existing .zshrc file with the template file."
```

Comment:
We don't have to echo this, so can we only have an if condition with `echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}` ?

Reply:
done